CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Braintree Android SDK Release Notes
 
+## unreleased
+
+* Remove Bintray dependency for Cardinal SDK (fixes #373)
```

Comment:
I think we should maybe add a note or something in bold explaining exactly what to remove from your gradle file.

Reply:
We added a link to the dev docs where merchants are told to add these credentials, and said they can now be removed.